src/plugins/data_source_management/public/components/create_data_source_wizard/components/header/header.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import {
+  EuiBetaBadge,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiLink,
+  EuiFlexItem,
+  EuiFlexGroup,
+} from '@elastic/eui';
+
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import { DocLinksStart } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../../../opensearch_dashboards_react/public';
+import { DataSourceManagementContext } from '../../../../types';
+
+export const Header = ({
+  isBeta = false,
+  docLinks,
+}: {
+  isBeta?: boolean;
+  docLinks: DocLinksStart;
+}) => {
+  const changeTitle = useOpenSearchDashboards<DataSourceManagementContext>().services.chrome
+    .docTitle.change;
+
+  const createDataSourceHeader = i18n.translate('dataSourcesManagement.createDataSourceHeader', {
+    defaultMessage: 'Create data source connection',
+  });
+
+  changeTitle(createDataSourceHeader);
+
+  return (
+    <EuiFlexGroup justifyContent="spaceBetween">
+      <EuiFlexItem grow={false}>
+        <div>
+          <EuiTitle>
+            <h1>
+              {createDataSourceHeader}
+              {isBeta ? (
```

Comment:
isBeta not needed for this feature as far as I know

Reply:
Done. Removed